Virtusize.podspec

```diff
@@ -11,9 +11,12 @@ Pod::Spec.new do |s|
   s.platform = :ios
   s.ios.deployment_target = '10.3'
   s.swift_version = '5'
-
+  
   s.source_files = ["Virtusize/Sources/*.{swift, h}", "Virtusize/Sources/**/*.swift"]
   s.resources = "Virtusize/Sources/Resources/**/*.otf"
   s.resource_bundle = { 'Virtusize' => ["Virtusize/Sources/Resources/**/*.lproj", "Virtusize/Sources/VirtusizeAssets.xcassets"] }
+
+  s.pod_target_xcconfig = { 'BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES' }
```

Comment:
There was some issue when the BUILD_LIBRARY_FOR_DISTRIBUTION flag was not set to YES for the Pods.
Add this config in the podspec to make sure when clients run `pod install`, the generated Pods targets have this flag set to Yes
<img width="1172" alt="Screen Shot 2021-11-18 at 12 17 20" src="https://user-images.githubusercontent.com/7802052/142345640-9179c70d-11bc-4a65-8ffd-900737554787.png">